src/app/services/login.service.js

```diff
@@ -139,8 +142,13 @@
     }
 
     function logout() {
+      $injector.get('networkService').logout({
```

Comment:
Shouldn't need to do this injection. If you put `networkService` in the function definition on line 9 it will automatically get the service and let this line just be:

```suggestion
      networkService.logout({
```

Reply:
There is a circular reference that this allows me to get around.  
![image](https://github.com/user-attachments/assets/ec5b59c0-817e-496f-ac18-5a0892836dfb)